### DISCUSSION TITLE
Como testar um código baixado no Github

### DISCUSSION THREAD

**Discussion by Vinikilzon** [frontendbr/forum#2479] [Perguntas e Respostas]
Olá, pessoal! Sou novo nesse mundo e achei um código aqui para um site que estou querendo montar. Baixei o .zip dele mas não sei como testo pra ver se está funcionando. Alguém poderia me ajudar?

**Comment by ghost** [comment]


**Reply by Vinikilzon** [reply]
seria esse script: 
https://github.com/RomaniukVadim/casino_script

Ele tem mais de uma linguagem.
